password_security/models/res_users.py

```diff
@@ -205,3 +205,13 @@ def _set_encrypted_password(self, uid, pw):
 
         self.write({"password_history_ids": [(0, 0, {"password_crypt": pw})]})
         return res
+
+    def action_reset_password(self):
+        """Disallow password resets inside of Minimum Hours"""
+        if not self.env.context.get("install_mode") and not self.env.context.get(
+            "create_user"
+        ):
+            if self.env.uid != SUPERUSER_ID:
```

Comment:
Whoops, this doesn't work, due to my bad suggestion. `SUPERUSER_ID` is the true root user, when really I should have said `self.env.ref('base.user_admin').id` for the admin user.

Reply:
Grepping the codebase a bit, I think `self.env.is_admin()` encapsulates this a bit better.